lib/fluent/plugin/kinesis_helper/client.rb

```diff
@@ -125,12 +127,15 @@ def setup_credentials
             credentials_options[:policy] = c.policy if c.policy
             credentials_options[:duration_seconds] = c.duration_seconds if c.duration_seconds
             credentials_options[:external_id] = c.external_id if c.external_id
+            credentials_options[:sts_endpoint_url] = c.sts_endpoint_url if c.sts_endpoint_url
             if c.sts_http_proxy and @region
                 credentials_options[:client] = Aws::STS::Client.new(region: @region, http_proxy: c.sts_http_proxy)
-            elsif @region
```

Comment:
Should we remain the case that neither of c.sts_http_proxy and c.sts_endpoint_url are set?

Reply:
Thanks!